### PR TITLE
style: adjust template of new PR

### PR DESCRIPTION
**Pull request by knst**
## Issue being fixed or feature implemented
Currently template for "New pull request" have a lot of instruction that are formatted as comments by html/markdown tag.
It makes this instruction to append in the final commit message when pull request is merged because they are invisible for reviewer (marked as comment).

With new template the messages would be more likely to be cleaned up before merging.

## What was done?
Replaced commented text in template to cursive test (italic)

## How Has This Been Tested?
By clicking "Preview" on GitHub

## Breaking Changes
No breaking changes, it's trivial change

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by ogabrielides — DISMISSED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,53 +1,52 @@
-<!--
-*** Please remove the following help text before submitting: ***
+***Please remove the following help written in cursive before merging***
```
Comment (thephez) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
***Please remove the italicized help prompts before submitting or merging***
```

**Review by thephez — CHANGES_REQUESTED** [review]
I didn't realize that was all included in the commit. I agree, it's probably best to remove it :+1: I'd just recommend a slight change in wording.

**Review by PastaPastaPasta — DISMISSED** [review]
utACK for squash merge. This will make it more likely they are removed.

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Review by thephez — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**